binarySearch/binarySearch.js

```diff
@@ -9,7 +9,56 @@
  *  * const index = binarySearch([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 2);
  *	* console.log(index); // 1
 **/
+// const binarySearch = (nums, target) => {
+//   function bSearch(nums, target, low, high) {
+//     if (high === low) {
+//       return low;
+//     }
+//     let mid = Math.ceil((low + high)/2);
+//     if (nums[mid] === target) {
+//       return mid;
+//     } else if (nums[mid] < target) {
+//      if (low === mid) {
+//        return false;
+//       else {
+//         return bSearch(nums, target, mid + 1, high);
+//       } 
+//     } if (nums[mid] > target) {
+//      if (high = mid) {
+//        return false;
+//       else {
+//         return bSearch(nums, target, low, mid - 1)
+//       }
+//     }
+//   }
+//   if (nums.length === 0) {
+//     return false;
+//   } else {
+//     return bSearch(nums, target, 0, nums.length - 1);
+//   }
+// };
 
 const binarySearch = (nums, target) => {
-
+  function bSearch(nums, target, low, high) {
```

Comment:
Very nice recursive approach here. I've seen Binary searches go both iteratively and recursively. personally I like reading recursive more.